src/logic/operate.js

```diff
@@ -0,0 +1,30 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    try {
+      return one.div(two).toString();
+    } catch (err) {
+      return "Can't divide by 0.";
+    }
```

Comment:
- [ ] [REQUIRED] Good job so far! The `error` is shown on the display when try to use `divide` operate option, please fix it 🙏🏼 

![mobile (3)](https://github.com/Peter-Dumbari/Maths-Magician/assets/63915024/04b9fa66-d642-469d-9d63-00131d8f6a97)